### PR TITLE
Inertia link `active-class` and `tag` props

### PR DESCRIPTION
**Pull request by digistorm**
Vue Router's `active-class` (https://router.vuejs.org/api/#active-class) and `tag` (https://router.vuejs.org/api/#tag) props are really useful.

This is a similar implementation.

I did not add a default value for the `active-class`, I'll leave that up to you if it's something you think should be included.

### CODE REVIEW

File: src/link.js [hunk 1]
```diff
@@ -27,14 +27,22 @@ export default {
       type: Boolean,
       default: false,
     },
+    tag: {
+      type: String,
+      default: 'a',
+    },
+    activeClass: {
+      type: String,
```
Comment (Juhlinus) [inline src/link.js]: Would we perhaps want to add a default value for this class, in similar fashion to what vue-router does?

> active-class
> type: string
> 
> default: "router-link-active"
> 
> Configure the active CSS class applied when the link is active. Note the default value can also be configured globally via the linkActiveClass router constructor option.

Maybe something along the lines of `inertia-link-active`? Or perhaps just `link-active`?

Comment (digistorm) [reply src/link.js]: Happy to add that, didn't feel like that's my decision to make.

**Comment by Larsklopstra** [comment]
Will this be implemented soon? Can't wait to use this in production

**Comment by DominikAger** [comment]
Will this be implemented soon?

File: src/link.js (hunk 1 above)
Comment (robsontenorio) [reply src/link.js:35]: This would make sense.

**Comment by Juhlinus** [comment]
@DominikAger Jonathan currently has his plate full of other things at the moment, but has mentioned that he aims to return to active Inertia.js development around June.

**Comment by Cannonb4ll** [comment]
This is a no-brainer to have this implemented just as vue-router does. Would be awesome 👍

**Comment by lostdesign** [comment]
@reinink do you need help with such reviews or any reason why PRs like these are held off for almost a year?

**Comment by reinink** [comment]
Hello, thanks so much for your interest in Inertia.js and for contributing back to it. However, in an effort to make the project easier to maintain, we've merged the four client-side libraries (`inertia`, `inertia-react`, `inertia-svelte` and `inertia-vue`) into a single [monorepo](https://github.com/inertiajs/inertia). If you feel this PR is still relevant, please resubmit it there. Sorry for the inconvenience. ❤️

**Pull request closed without merging**